distribution/changelog.txt

```diff
@@ -4,10 +4,11 @@
 - Feature: [#18732] [Plugin] API to get the guests thoughts.
 - Feature: [#18744] Cheat to allow using a regular path as a queue path.
 - Feature: [#19023] Add Canadian French translation.
+- Feature: [#19276] Add Powered Lifthill to Giga Coaster.
 - Feature: [#19378] Add command to combine CSG1i.DAT and CSG1.DAT.
 - Feature: [objects#226] Port RCT1 Corkscrew Coaster train.
 - Feature: [objects#229] Port RCT1 go karts with helmets.
-- Improved: [#11473] Hot reload for plug-ins now works on macOS.
+- Improved: [#11473] Hot eload for plug-ins now works on macOS.
```

Comment:
Accidental change.

Reply:
Note to self, don't try rebasing in browser on phone.

Reply:
The GitHub functionality does a merge, rather than a rebase. It’s fine for very simple PRs that get squished anyway, for anything else it’s better to use a proper computer.